Project.toml

```
@@ -25,15 +30,18 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 BetterFileWatching = "^0.1.2"
 Configurations = "0.16"
 FromFile = "0.1"
+Distributions = "0.25"
 Git = "1"
 GitHubActions = "0.1"
 HTTP = "^0.9.3"
-Pluto = "0.15, 0.16, 0.17"
+OrderedCollections = "1"
+Pluto = "0.17.3"
```

```suggestion
Pluto = "=0.17.3"
```
Without the `=`, any version of `0.17` can be taken.

---

I wanted `0.17.3 - 0.17.99`